spec.bs

```diff
@@ -297,7 +334,8 @@ To <dfn>register an import map</dfn> given an [=environment settings object=] |s
   1. If |parsed|["`scopes`"] [=map/exists=], then:
     1. If |parsed|["`scopes`"] is not a [=map=], then throw a {{TypeError}} indicating that the "`scopes`" top-level key must be a JSON object.
     1. Set |sortedAndNormalizedScopes| to the result of [=sorting and normalizing scopes=] given |parsed|["`scopes`"] and |baseURL|.
-  1. Return the [=/import map=] whose [=import map/imports=] are |sortedAndNormalizedImports| and whose [=import map/scopes=] scopes are |sortedAndNormalizedScopes|.
+  1. Let |import map| be a new [=/import map=] whose [=import map/imports=] are |sortedAndNormalizedImports| and whose [=import map/scopes=] scopes are |sortedAndNormalizedScopes|.
+  1. Return an [=import map parse result=] with [=import map parse result/settings object=] is ... [=import map parse result/import map=] is |import map|, and [=import map parse result/error to rethrow=] is ....
```

Comment:
Probably this would work better as a wrapper algorithm that catches any errors thrown by "parse an import map"

Reply:
Introduced `create an import map parse result`.